tests/unit/test_sql_sensor.py

```diff
@@ -110,8 +110,7 @@ def tearDownClass(cls) -> None:
 
 class ClickHouseLegacySqlSensorTestCase(unittest.TestCase):
     @mock.patch(
-        'airflow_clickhouse_plugin.ClickHouseSqlSensor'
-            '._ClickHouseSqlSensor__poke_clickhouse',
+        'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor.ClickHouseSqlSensor'
     )
```

Comment:
Not sure if this patch is needed anymore. Does this test fail if we drop that `mock.path` call?

Reply:
I've removed it and it was failing because the method call in the body of the test. I've removed the assert inside and everything got fixed :)